### PR TITLE
holesky support

### PR DESCRIPTION
**Pull request by lior-blox**

### CODE REVIEW

File: eth1_deposit/eth1_deposit_test.go [hunk 1]
```diff
@@ -86,10 +86,65 @@ func TestPraterDepositData(t *testing.T) {
...
-			expectedSig:                   _ignoreErr(hex.DecodeString("a2bcc9d2ac82062cb9806b761e8e8d405963620b8f5356fa70fe543812bf07c3031546482c737401ba1dec01d5690d0600c900ebe7dca5699e804ff4441ed4e25789b389bcdc69c6f4dc25ef40e5694f6de7723bda359c5c2a54e05ae90290ca")),
-			expectedRoot:                  _ignoreErr(hex.DecodeString("d243130779e16b4352bb8d2c80765334b4a7bdd4bc42356b37e42380dc47dac5")),
+			withdrawalPubKey:              _ignoreErr(hex.DecodeString("8d176708b908f288cc0e9d43f75674e73c0db94026822c5ce2c3e0f9e773c9ee95fdba824302f1208c225b0ed2d54154")),
+			expectedWithdrawalCredentials: _ignoreErr(hex.DecodeString("005b55a6c968852666b132a80f53712e5097b0fca86301a16992e695a8e86f16")),
+			expectedSig:                   _ignoreErr(hex.DecodeString("a88d0fd588836c5756ec7f2fe2bc8b6fc5723d018c8d31c8f42b239ac6cf7c2f9ae129caafaebb5f2f25e7821678b41819bc24f6eeebe0d8196cea13581f72ac501f3e7e9e4bc596e6a545ac109fb2ff1d7eb03923454dc5258718b43427a757")),
+			expectedRoot:                  _ignoreErr(hex.DecodeString("aa940a26af67a676bcd807b0fd3f39aadbfc6862e380e115051683e1fccc0171")),
```
Comment (lior-blox) [inline eth1_deposit/eth1_deposit_test.go:92]: @moshe-blox @GalRogozinski can you guys double check this please.
IMO we didn't fix this for prater in the last hotfix PR

**Review by MatheusFranco99 — APPROVED** [review]
Double-checked the values. Looks good to me

File: eth1_deposit/eth1_deposit_test.go (hunk 1 above)
Comment (GalRogozinski) [reply eth1_deposit/eth1_deposit_test.go:92]: I don't understand why `Prater` test should change...
Anyhow if the test passes I guess everything is fine assuming VerifyOperation does sig check

Comment (lior-blox) [reply eth1_deposit/eth1_deposit_test.go:92]: Cause it did not match the eth Launchpad.

Comment (GalRogozinski) [reply eth1_deposit/eth1_deposit_test.go:92]: what do you mean didn't match the eth launchpad? aren't those credentials different for every user?

**Review by GalRogozinski — APPROVED** [review]

File: eth1_deposit/eth1_deposit_test.go (hunk 1 above)
Comment (lior-blox) [reply eth1_deposit/eth1_deposit_test.go:92]: @GalRogozinski as we discussed.
I aligned that all input values are the same between all the different networks.
So only the root and sig are expected to be different.

Comment (moshe-blox) [reply eth1_deposit/eth1_deposit_test.go:92]: just for the record: this isn't a requirement of the test, but it helps verify that different Eth networks yield different roots & sigs

**Review by moshe-blox — APPROVED** [review]

**Pull request merged**